config/jobs/kubernetes/test-infra/test-infra-periodics.yaml

```diff
@@ -36,6 +36,9 @@ periodics:
       env:
       - name: GOOGLE_APPLICATION_CREDENTIALS
         value: /etc/service-account/service-account.json
+      # Go incorrectly determines the number of CPUs in a pod, set manually to (2*CPUs-1) corresponding to test-infra-periodics.yaml
```

Comment:
this comment is now amusingly self-referential, but I'd prefer to get this fix merged, as it's not overly harmful, just amusing, and triage has been down for a while

Reply:
Why did it suddenly stop working?